Sources/SwifterSwift/SwiftStdlib/BinaryIntegerExtensions.swift

```diff
@@ -0,0 +1,49 @@
+// BinaryIntegerExtensions.swift - Copyright 2021 SwifterSwift
+
+#if canImport(Foundation)
+import Foundation
+
+// MARK: - Properties
+
+public extension BinaryInteger {
+    /// SwifterSwift: The raw bytes of the integer.
+    ///
+    ///     var number = Int16(-128)
+    ///     print(number.bytes)
+    ///     // prints "[255, 128]"
+    ///
+    var bytes: [UInt8] {
+        var result = [UInt8]()
```

Comment:
Nit: Since we know the size of result will be `MemoryLayout<Self>.size` maybe `reserveCapacity` could be a small speedup